Resources/GameData/kOS/Parts/kOSMachineRad/part.cfg

```diff
@@ -0,0 +1,71 @@
+PART
+{
+// --- general parameters ---
+name = kOSMachineRad
+module = Part
+author = Peter Goddard
+
+// --- asset parameters ---
+mesh = model/model.mu
+scale = 1
+rescaleFactor = 0.99999999999
+iconCenter = 0, 0, 0
+
+// --- node definitions ---
+node_attach = 0.0, 0.0, 0.0, 1, 0, 0
+
+// --- Tech tree ---
+TechRequired = flightControl
+
+// --- editor parameters ---
+cost = 2200
+entryCost = 4200
+category = Control
+subcategory = 0
+title = CompoMax Radial Tubeless
+manufacturer = Squalid-State Devices Inc.
+description = Would you trust life and limb to a mindless autopilot, powered by untested software you hastily wrote yourself? Spacefaring kerbals would!
+
+// attachment rules: stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+attachRules = 0,1,0,0,1
+
+// --- standard part parameters ---
+mass = 0.00001
+dragModelType = default
+maximum_drag = 0.0
+minimum_drag = 0.0
+angularDrag = 0
+crashTolerance = 6
+maxTemp = 3400
+
+
+MODULE
+{
+	name = kOSProcessor
+	diskSpace = 60000

```

Comment:
60k is quite a bit bigger than our other processors. It is designed to be an end game part?